roles/users.nix

```diff
@@ -18,18 +18,4 @@ in
 
   # No mutable users
   users.mutableUsers = false;
-
-  # Assign keys from all users in wheel group
-  # This is only done because nixops cant be deployed from any other account
-  users.extraUsers.root.openssh.authorizedKeys.keys = lib.unique (
```

Comment:
So effectively you are disabling the root login via ssh? We are still depending on it here: https://github.com/nix-community/infra/blob/6cea644066371ce3bcd107ad3a49bd4bc2b3679a/tasks.py#L21

Reply:
Dropped this commit.